pod/perldiag.pod

```diff
@@ -3345,6 +3351,14 @@ The number of items in a hash can be obtained by doing:
 
     scalar(keys %hash);
 
+=item lexically_export can only be called at compile time
+
+(F) A call to L<builtin/lexically_export> creates new names in the lexical
+scope of the code currently being compiled.  This function can therefore
+only be called when a scope is being compiled (i.e. from a C<BEGIN> block
+in that scope).  If there is no code currently being compiled, this function
+will have no effect and so calling it is not permitted.
```

Comment:
Same here. How about something like this?

> (F) L<builtin/lexically_export> was called at runtime. Because it creates new names in the lexical scope
> currently being compiled, it can only be called from a C\<BEGIN\> block in that scope.